spec/19-grammar.md

```diff
@@ -849,6 +849,7 @@ The grammar notation is described in [Grammars section](09-lexical-structure.md#
 
 <i id="grammar-breakout-level">breakout-level:</i>
    <i><a href="#grammar-integer-literal">integer-literal</a></i>
+   <i>(   <a href="#grammar-integer-literal">integer-literal</a>   )</i>
```

Comment:
Should also be updated in `11-statements.md`
https://github.com/php/php-langspec/blob/1baff2105284c1e964de18fc7d40d7b1dc6900a0/spec/11-statements.md

Reply:
I'm on it